Guidance on how to retry messages using the native tools provided (#5539)
MassTransit/MassTransit: doc/content/3.documentation/6.transports/3.azure-service-bus.md

![Fault topology for Azure Service Bus](/azure-topology-fault.svg)

Go to [Exceptions to learn more on exception and faults](/documentation/concepts/exceptions)

## Retrying messages

> The Azure Service Bus portal provides a method to retry failed messages.

Reviewer: ```suggestion
The Azure Service Bus Portal provides a method to retry failed messages by doing the following:
```

1. Open the Service Bus namespace

> 2. Select the queue that needs to be reviewed

Reviewer: ```suggestion
2. Select the queue that has failed messages
```
